hyper/contrib.py

```diff
@@ -43,28 +48,40 @@ def get_connection(self, host, port, scheme, cert=None):
         if cert is not None:
             ssl_context = init_context(cert=cert)
 
+        connection_key = (host, port, scheme, cert, proxy)
```

Comment:
At this point we have enough commits that you don't need to worry about reverting: we'll need to squash before we merge to clean them up anyway. :smile: So just make the change directly to get the code looking the way you want it. :grin: